http4k-contract/src/main/kotlin/org/http4k/contract/routeMeta.kt

```diff
@@ -102,6 +120,22 @@ class RouteMetaDsl internal constructor() {
         receiving(RequestMeta(Request(POST, "").with(body.first of body.second), definitionId, body.second, schemaPrefix))
     }
 
+    /**
+     * Add negotiated example requests to this route.  It is also possible to pass in the definitionId
+     * for this request body which will override the naturally generated one.
+     */
+    @JvmName("receivingNegotiated")
+    fun <T> receiving(
+        negotiated: Pair<AutoContentNegotiator<T>, T>,
+        definitionId: String? = null,
+        schemaPrefix: String? = null
+    ) {
+        for (lens in negotiated.first) {
+            receiving(lens to negotiated.second, definitionId, schemaPrefix)
+        }
+        requestBody = negotiated.first.toBodyLens()
```

Comment:
`RouteMeta` can only support a single body lens.  I attempted to convert it to a list, but ran into issues.  This compromise isn't perfect, but seems to work well enough.